docs/java/java-spring-cloud.md

```diff
@@ -0,0 +1,167 @@
+---
+Area: java
+TOCTitle: Java Spring Cloud
+ContentId: 
+PageTitle: Build and Deploy Java Spring Boot Apps to Azure Spring Cloud with Visual Studio Code
+DateApproved: 
+MetaDescription: Java Spring app tutorial showing how to build and deploy a Java Spring Boot microservices to Azure Spring Cloud with Visual Studio Code
+---
+
+# Java Spring Cloud with Visual Studio Code
+
+This tutorial shows you how to create a Java web application with Visual Studio Code. You'll learn how to run, debug, and edit the Java web app locally and eventually on a fully managed Microservices platform built for Java workloads: [Azure Spring cloud](https://azure.microsoft.com/services/spring-cloud/).
+
+## Scenario
+
+We will deploy a simple Spring Boot Getting Started web app to Azure Spring Cloud.
+
+Azure Spring Cloud makes it easy to deploy Spring Boot microservice applications to Azure without any code changes. The service manages the infrastructure of Spring Cloud applications so developers can focus on their code. Other benefits include:
+
+- Efficiently migrate existing Spring apps and manage cloud scaling and costs.
+- Modernize apps with Spring Cloud patterns to improve agility and speed of delivery.
+- Run Java at cloud scale and drive higher usage without complicated infrastructure.
+- Develop and deploy rapidly without containerization dependencies.
+- Monitor production workloads efficiently and effortlessly.
+
+![Greeting from Java](images/java-webapp/greeting-from-spring.png)
+
+## Before you begin
+
+Before running and deploying this sample, you must have the Java SE Development Kit (JDK) and Apache Maven build tools on your local development environment. If you don't have, please install them.
+
+Download and install the Java SE Development Kit (JDK), version 8:
```

Comment:
JDK version need be specified more clearly. VS Code Java itself requires JDK 11 or above version, so to successfully follow this tutorial, JDK 11 or above is needed. As we suggest user to use extension pack, JDK 11 is included as part of the pack and no need for user to handle it separately. Can we list using Extension Pack as a prerequisite of this tutorial to replace the "Download and install the Java SE Development Kit (JDK), version 8:"?

Reply:
done

Reply:
Thanks. @gregvanl , the PR looks good to me. I think it can be merged.